R/routes.R

```diff
@@ -80,7 +80,7 @@
 #' route_cyclestreet(c(-2, 52), c(-1, 53), "fastest")
 #' }
 route_cyclestreet <- function(from, to, plan = "fastest", silent = TRUE, pat = cyclestreet_pat(),
-                              base_url = "http://www.cyclestreets.net/api/", reporterrors = FALSE){
+                              base_url = "http://www.cyclestreets.net", reporterrors = FALSE){
```

Comment:
Curious how often these base URLs change?  If they do change frequently enough then make sense to pass in as a param, but if they change maybe once a year, then hard coding within the function or elsewhere in the pkg simplifies the function. But it could make sense here, it's up to you. Like maybe you need to pass in a different URL sometimes, like a URL for a test server, eg. ,